.github/workflows/ci.yml

```diff
@@ -101,15 +100,15 @@ jobs:
               uses: actions/checkout@v4
 
             - name: Install povray
-              run: sudo apt-get install povray
+              run: sudo apt-get update && sudo apt-get install povray
```

Comment:
`povray` install started failing transiently with various HTTP errors. Not sure if this will help, we'll see.